### PR TITLE
Add checkadmin middleware template

### PR DESCRIPTION
**Pull request by hoangtm1601**
add checkadmin middleware, category crud

### CODE REVIEW

**Comment by hoangtm1601** [comment]
ready

File: app/Http/Controllers/CategoryController.php [hunk 1]
```diff
@@ -24,24 +47,35 @@ public function index()
...
     {
-        //
+        $validatedData = $request->validated();
+        $recentlyAddedCategory = $this->categoryRepository->create($validatedData);
+        if ($recentlyAddedCategory) {
+            flash(__('Add successfully'))->success();
+            return redirect()->route('categories.index');
+        } else {
+            flash(__('Add fail'))->error();
+            return redirect()->route('categories.index');
```
Comment (BunnyPi04) [inline app/Http/Controllers/CategoryController.php]: thiếu dòng trắng trc return

File: routes/web.php [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
-            'prefix' => 'admin'
-        ],
-        function () {
-            Route::get('/', 'AdminController@index')->name('admin.index');
-        }
-    );
+Route::group(
+    [
+        'prefix' => 'admin',
+        'middleware' => 'auth'
```
Comment (BunnyPi04) [inline routes/web.php]: https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_27-mang-va-chuoi-12

**Comment by hoangtm1601** [comment]
ready

**Pull request merged**